packages/runtime/runtime-utils/src/summaryUtils.ts

```diff
@@ -130,18 +130,6 @@ export function addBlobToSummary(
 	summary.stats.totalBlobSize += getBlobSize(content);
 }
 
-/**
- * @internal
- */
-export function addTreeToSummary(
```

Comment:
Completely identical to `addSummarizeResultToSummary` below

Reply:
Is it worth adding a changeset for this? Or does is not matter since this is marked `@internal`

Reply:
Yep shouldn't matter since it's internal